test/built-ins/RegExp/unicode_character_class_backspace_escape.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-ClassEscapes
+description: >
+  \b escape inside CharacterClass is valid in Unicode patterns (unlike \B).
+info: |
+  ClassEscape[U] ::
+    b
+
+  Static Semantics: CharacterValue
+
+  ClassEscape :: b
+
+  1. Return the code point value of U+0008 (BACKSPACE).
+---*/
+
+assert(/[\b]/u.test('\b'));
```

Comment:
You could imagine a broken engine that treats `\b` as `b` both in string literals and in `u` RegExps. Such an engine would still pass this test. How about:

```js
const backspace = '\b';
assert(backspace.charCodeAt() === 0x08);
assert(/[\b]/u.test(backspace));
```